tests/symengine/symengine_tensor_operations_02.output

```diff
@@ -29,26 +29,26 @@ DEAL:Integer:Copy constructor::cs: 5 7 7 6
 DEAL:Integer:Math operators:Tensor::c+=b: 14 16 18 20
 DEAL:Integer:Math operators:Tensor::c*=s: 28 32 36 40
 DEAL:Integer:Math operators:Tensor::c-=b: 19 22 25 28
-DEAL:Integer:Math operators:Tensor::c/=s: 19/2 11 25/2 14
+DEAL:Integer:Math operators:Tensor::c/=s: 9.5 11.0 12.5 14.0
```

Comment:
ha, that's actually quite funny that that worked!

Reply:
No, I think that this output is now incorrect. Division where the numerator and demoninator are both integers should result in rational type.

Reply:
Note: You're not seeing arithmetic types being printed here. These are SymEngine types being printed with their own overloaded operator<<.

Reply:
Drats, so this implies that the `Number(1.)` in
```
const Number inverse_factor = Number(1.) / factor;
```
needs to change to `Number(1)` else when `Number` is a `SD::Expression`

Reply:
We should be able to just use the old implementation for `Differentiation::SD::Expression`.

Reply:
Sounds fine with me.